### PR TITLE
MSA Upgrade workflow for AI ML

### PR DESCRIPTION
**Pull request by amitraj83**

### CODE REVIEW

File: Upgrade_MS_Activator/Upgrade_MS_Activator.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<ObjectDefinition>
+  <variables frozen="0">
+    <variable displayName="host" name="params.host" startIncrement="0" type="String" mandatoryArray="false" visible="true" description="IP address of the target host" groupSeparator="" groupDisplayName="" displayOrder="0" increment="0" refServiceURI="" keepOnImport="false" editable="false" onlyDetailView="false" localVarNameMatch="" remoteVarNameMatch="" arrayCanAdd="true" arrayCanRemove="true" arrayCanMove="true" arrayCanEdit="true" displayNameHeader="" fullDisplayName="" isMandatory="false" isUserLocked="false" isGrouped="false" isSearchable="false" isUniqueGlobal="false"/>
```
Comment (abr-ubiqube) [inline Upgrade_MS_Activator/Upgrade_MS_Activator.xml:3]: @amitraj83 
can you use either always upper case for first character of variable display name or always lower case ?

File: Upgrade_MS_Activator/Upgrade_MS_Activator.xml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    <visibility>5</visibility>
+    <allowSchedule/>
+    <task name="Task_rundockercompose.py">
+      <processPath>/opt/fmc_repository/Process/workflows/Upgrade_MS_Activator/Process_Run_Docker_Compose/Tasks</processPath>
+      <displayName>rundockercompose</displayName>
+    </task>
+  </process>
+  <information>
+    <icon/>
+    <description>Upgrades the MS Activator to the latest version</description>
```
Comment (abr-ubiqube) [inline Upgrade_MS_Activator/Upgrade_MS_Activator.xml:41]: MS Activator -> MSActivator

**Review by abr-ubiqube — CHANGES_REQUESTED** [review]
please update "MS Activator" to "MSActivator"

File: Upgrade_MS_Activator/.meta_Upgrade_MS_Activator.xml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<metadata>
```
Comment (abr-ubiqube) [inline Upgrade_MS_Activator/.meta_Upgrade_MS_Activator.xml:2]: Meta file is incorrect

**Review by abr-ubiqube — CHANGES_REQUESTED** [review]

File: Upgrade_MS_Activator/Upgrade_MS_Activator.xml (hunk 2 above)
Comment (amitraj83) [reply Upgrade_MS_Activator/Upgrade_MS_Activator.xml:41]: ok

File: Upgrade_MS_Activator/.meta_Upgrade_MS_Activator.xml (hunk 3 above)
Comment (amitraj83) [reply Upgrade_MS_Activator/.meta_Upgrade_MS_Activator.xml:2]: I think I will add the content manually because I believe its a bug

Comment (amitraj83) [reply Upgrade_MS_Activator/.meta_Upgrade_MS_Activator.xml:2]: Done

File: Upgrade_MS_Activator/Upgrade_MS_Activator.xml (hunk 2 above)
Comment (amitraj83) [reply Upgrade_MS_Activator/Upgrade_MS_Activator.xml]: Done

File: Upgrade_MS_Activator/Upgrade_MS_Activator.xml (hunk 1 above)
Comment (amitraj83) [reply Upgrade_MS_Activator/Upgrade_MS_Activator.xml]: Done

**Pull request closed without merging**